<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFeatureTagPropertySettingDialog</class>
 <widget class="QDialog" name="CFeatureTagPropertySettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>468</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>361</width>
    <height>505</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QGroupBox" name="vGroupBox">
     <property name="title">
      <string>标签位置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QRadioButton" name="vTopRadioButton">
          <property name="text">
           <string>顶端</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>偏移量</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="vTopOffsetLineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="vMiddleRadioButton">
        <property name="text">
         <string>居中</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QRadioButton" name="vBottomRadioButton">
          <property name="text">
           <string>底端</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>偏移量</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="vBottomOffsetLineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="lineGroupBox">
     <property name="title">
      <string>标签线</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QCheckBox" name="lineVisibleCheckBox">
          <property name="text">
           <string>显示</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>颜色</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="lineColorButton">
          <property name="minimumSize">
           <size>
            <width>92</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>92</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>线宽</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="lineWidthSpinBox">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>120</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>线型</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="lineStyleComboBox">
          <property name="minimumSize">
           <size>
            <width>69</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>69</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="hGroupBox">
     <property name="title">
      <string>标签线位置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="HLeftRadioButton">
          <property name="text">
           <string>居左</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>偏移量</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="HLeftOffsetLineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="HMiddleRadioButton">
        <property name="text">
         <string>居中</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="HRightRadioButton">
          <property name="text">
           <string>居右</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>偏移量</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="HRightOffsetLineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="bottomPointGroupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>底端端点</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QCheckBox" name="bottomPointVisibleCheckBox">
          <property name="text">
           <string>显示</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>颜色</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bottomPointColorButton">
          <property name="minimumSize">
           <size>
            <width>92</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>92</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>尺寸</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="bottomPointSizeSpinBox">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>类型</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="bottomPointStyleComboBox">
          <property name="minimumSize">
           <size>
            <width>69</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>69</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="topPointGroupBox">
     <property name="title">
      <string>顶端端点</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QCheckBox" name="topPointVisibleCheckBox">
          <property name="text">
           <string>显示</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>颜色</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="topPointColorButton">
          <property name="minimumSize">
           <size>
            <width>92</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>92</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>尺寸</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="topPointSizeSpinBox">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_11">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>类型</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="topPointStyleComboBox">
          <property name="minimumSize">
           <size>
            <width>69</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>69</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>vTopRadioButton</tabstop>
  <tabstop>vTopOffsetLineEdit</tabstop>
  <tabstop>vMiddleRadioButton</tabstop>
  <tabstop>vBottomRadioButton</tabstop>
  <tabstop>vBottomOffsetLineEdit</tabstop>
  <tabstop>HLeftRadioButton</tabstop>
  <tabstop>HLeftOffsetLineEdit</tabstop>
  <tabstop>HMiddleRadioButton</tabstop>
  <tabstop>HRightRadioButton</tabstop>
  <tabstop>HRightOffsetLineEdit</tabstop>
  <tabstop>lineVisibleCheckBox</tabstop>
  <tabstop>lineColorButton</tabstop>
  <tabstop>lineWidthSpinBox</tabstop>
  <tabstop>lineStyleComboBox</tabstop>
  <tabstop>topPointVisibleCheckBox</tabstop>
  <tabstop>topPointColorButton</tabstop>
  <tabstop>topPointSizeSpinBox</tabstop>
  <tabstop>topPointStyleComboBox</tabstop>
  <tabstop>bottomPointVisibleCheckBox</tabstop>
  <tabstop>bottomPointColorButton</tabstop>
  <tabstop>bottomPointSizeSpinBox</tabstop>
  <tabstop>bottomPointStyleComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>vTopRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>vTopOffsetLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HLeftRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>HLeftOffsetLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>665</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HRightRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>HRightOffsetLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>454</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>623</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vBottomRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>vBottomOffsetLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
